src/utils/utils.ts

```diff
@@ -106,3 +106,13 @@ export const isShortcutPressed = (pressedKeys: Set<string>, shortcutKeys: string
     });
   });
 };
+
+// we don't send case-level information, only info about dataContexts, collections + attributes
+// documentation about CODAP Data Interactive API notifications: https://github.com/concord-consortium/codap/wiki/CODAP-Data-Interactive-Plugin-API#codap-initiated-actions
+export const notificationsToIgnore = [
```

Comment:
Would it make sense to put this in src/constants.ts?

Reply:
Done.